src/pyk/ktool/krun.py

```diff
@@ -147,6 +151,8 @@ def run_kore_config(
         expand_macros: bool = False,
         bug_report: Optional[BugReport] = None,
         expect_rc: int = 0,
+        search_final: bool = False,
+        no_pattern: bool = False,
```

Comment:
```suggestion
        search_final: bool = False,
        no_pattern: bool = False,
        # ---
        bug_report: Optional[BugReport] = None,
        expect_rc: int = 0,
```

Here, `# ---` separates kast CLI parameters from the rest.